src/Native/Unix/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ add_compile_options(-Wno-old-style-cast)
 add_compile_options(-Wno-cast-align)
 add_compile_options(-Wno-typedef-redefinition)
 add_compile_options(-Wno-unused-local-typedef)
+add_compile_options(-Wno-enum-compare)
```

Comment:
I would prefer fixing the compares by casting the enum values to ints before comparison instead. I consider disabling the warning globally to be the last resort.

Reply:
IMO, those asserts are for sanity check. It was this one liner or casting those enum lines in multiple files. It's not a very useful warning, in fact less useful than `-Wno-cast-align` which we are ignoring on line 19. But I don't have strong opinion.

Reply:
But this disables the warnings for all the files. So if someone makes a mistake elsewhere and compares really unrelated enums, it would go unnoticed.